src/Helpers/RecognizeClassHelper.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\BLT\Helpers;
+
+use Illuminate\Support\Str;
+
+class RecognizeClassHelper
+{
+    public static function recognizeObjectClass(string $objectName): string
```

Comment:
Please think how it would work if Laravel application would be bigger than simple CRUD. For example:

```
+ app
    + Api
        + Http
        + Models
        + Providers
    + Client
        + Http
        + Models
    + Dashboard
        + Http
        + Models
        + Providers
    + Marketing
        + Http
        + Providers
+ bootstrap/
    (...)
(...)
```

Or even more complicated?

```
+ app
    + Console
    + Eloquent
        + Casts
        + Observers
        + Mutators
        Article.php
        User.php
+ bootstrap
    (...)
(...)
```